org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -369,12 +369,15 @@ Resources:
               - 'aws-foundational-security-best-practices/v/1.0.0/IAM.6'
               - 'arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/2.7'
               - 'cis-aws-foundations-benchmark/v/1.4.0/3.7'
-              # SecHub Recommendation: no security group allows unrestricted ingress access to port 22.
+              # SecHub Recommendation: no security group allows unrestricted ingress access to port 22/3389.
               # Supression Reason: Allow for bastian hosts and testing
               - 'cis-aws-foundations-benchmark/v/1.2.0/rule/4.1'
+              - 'arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/4.1'
```

Comment:
if these are ARNs then shouldn't `cis-aws-foundations-benchmark/v/1.4.0/3.7` be an ARN as well? or should we just remove that entry?

Reply:
That's what I would have thought, but there are some findings that have generatorID like 'cis-aws-foundations-benchmark/v/1.4.0/2.1.5.2' (i.e. not ARNs) so I'm not sure we can remove them (or rename them as ARNs). I also had questions about the version of CIS... The generatorIDs above refer to CIS v1.2.0 but now're we're following CIS v1.4.0... I looked for v1.4.0 versions of the generators but did not see any in the findings so I'm just leaving everything as-is (if something needs to change, we'll see new findings pop up.).